airbyte-integrations/connector-templates/source-configuration-based/source_{{snakeCase name}}/{{snakeCase name}}.yaml.hbs

```diff
@@ -33,3 +33,18 @@ streams:
 check:
   stream_names:
     - "customers"
+
+spec:
+  documentation_url: https://docsurl.com
+  connection_specification:
```

Comment:
It's worth noting that this went from camel case to snake case. I like better consistency with our component definitions across the file and the response of a `spec` call will still be camel case to retain compatibility.

However, it does make it a small gotcha for migrating existing connectors that it needs to be switched, but it will be very clear and obvious since the YAML will fail validation and report that the fields are missing